open-machine-learning-jupyter-book/assignments/ml-advanced/clustering/k-means-clustering.md

```diff
@@ -0,0 +1,10 @@
+# Try different clustering methods
+
+## Instructions
+
+In this lesson you learned about K-Means clustering. Sometimes K-Means is not appropriate for your data. Create a notebook using data either from these lessons or from somewhere else (credit your source) and show a different clustering method NOT using K-Means. What did you learn? 
+## Rubric
```

Comment:
Add acknowledge to all the new files you added. https://open-academy.github.io/machine-learning/assignments/data-science/small-diabetes-study.html#acknowledgments